samples/charts/category-chart/annotations/BlazorClientApp.sln

```diff
@@ -1,4 +1,4 @@
-
+﻿
```

Comment:
you must have some VS Code extension that keeps inserting not needed blank lines in .sln files